### PR TITLE
PXB-2369 Issues with "Installing" page in XtraBackup documentation (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
Merged 5.7 version
Modified source/installation

### CODE REVIEW

**Review by nastena1606 — APPROVED** [review]
LGTM

File: storage/innobase/xtrabackup/doc/source/installation.rst [hunk 1]
```diff
@@ -71,13 +73,13 @@ Binary tarballs are available for download and installation.
...
-Select the |Percona XtraBackup| 8.0 version number and the type of tarball for your installation. Both binary tarballs support all distributions.
+Selecting a different software, such as Ubuntu 20.04 (Focal Fossa), provides a tarball for that operating system. You can download the packages together or separately. 
 
-Fetch and extract the correct binary tarball. For example, the following downloads the full tarball:
+The following link is an example of downloading the full tarball for Linux/Generic:
 
 .. code-block:: bash
 
-  $ wget https://www.percona.com/downloads/Percona-XtraBackup-8.0/LATEST/percona-xtrabackup-8.0.13-Linux.x86_64.glibc2.12.tar.gz
+  $ wget https://downloads.percona.com/downloads/Percona-XtraBackup-LATEST/Percona-XtraBackup-8.0.23-16/binary/tarball/percona-xtrabackup-8.0.23-16-Linux-x86_64.glibc2.12.tar.gz
```
Comment (hrvojem) [inline storage/innobase/xtrabackup/doc/source/installation.rst:83]: 2.12 is for CentOS6, so 2.17 might be a better example for other distros (or you can add both)

**Review by hrvojem — APPROVED** [review]
LGTM with a minor comment

File: storage/innobase/xtrabackup/doc/source/installation.rst (hunk 1 above)
Comment (patrickbirch) [reply storage/innobase/xtrabackup/doc/source/installation.rst:83]: Fixed to 2.17

**Pull request merged**